websites/D/Discord Türkiye/presence.ts

```diff
@@ -0,0 +1,115 @@
+const presence = new Presence({ clientId: "890757020393816064" });
+
+presence.on("UpdateData", async () => {
+	const presenceData = {
+			largeImageKey: "1024dctr",
+			details: "Geziniyor",
+			state: "",
+			startTimestamp: Math.floor(Date.now() / 1000),
+			smallImageText: "",
+			smallImageKey: "",
+		},
+		page = document.location.pathname,
```

Comment:
change the variable name accordingly
```suggestion
		{ pathname } = document.location,
```